snuba/admin/clickhouse/system_queries.py

```diff
@@ -124,7 +55,7 @@ def run_system_query_on_host_by_name(
         (FROM|from)
         \s
         system.[a-z_]+
-        (?P<extra>\s[\w\s,=()*+<>'%\-\/]+)?
+        (?P<extra>\s[\w\s,=()*+<>'"%\-\/]+)?
```

Comment:
needed this to be able to have queries that include things like `partition = '(90,"2022-01-24")' `